Pipfile

```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 python-gnupg = "*"
 click = "*"
 requests = "*"
+gpg = "*"
```

Comment:
This library doesn't exist. It should be removed:
```suggestion

```

Reply:
Got it!